extras/dagger/main.go

```diff
@@ -388,6 +391,9 @@ func (att *Attestation) Push(
 	if passphrase != nil {
 		container = container.WithSecretVariable("CHAINLOOP_SIGNING_PASSWORD", passphrase)
 	}
+	if exceptionBypassPolicyCheck != nil && *exceptionBypassPolicyCheck {
```

Comment:
very nit: this is basically the same as not using a pointer and just checking for its `true` value

Reply:
I think it needs to be a pointer because it's marked as `optional` for dagger